doc/source/examples/unresolved-cfd-dem/single-particle-sedimentation/single-particle-sedimentation.rst

```diff
@@ -0,0 +1,362 @@
+
+The gravity is set to :math:`0` since the DEM simulation only loads the particle.
+
+.. code-block:: text
+
+    subsection lagrangian physical properties
+      set g                        = 0.0, 0.0, 0.0
+      set number of particle types = 1
+      subsection particle type 0
+        set size distribution type = uniform
+        set diameter               = 0.002663
+        set number of particles    = 1
+        set density particles      = 1029
+      end
+    end
+
+
+Insertion Info
+~~~~~~~~~~~~~~~~~~~
+
+We use the list insertion method to insert a single particle in our domain at a specific location:
+
+.. code-block:: text
+
+    subsection insertion info
+      set insertion frequency = 1
+      set insertion method    = list
+      set list x              = 0
+      set list y              = 0.08
+      set list z              = 0
+    end
+
+
+---------------------------
+Running the DEM Simulation
+---------------------------
+Assuming that the ``lethe-particles`` executable is within your path, the simulation can be launched on a single processor by typing:
+
+.. code-block:: text
+  :class: copy-button
+
+  lethe-particles initial-particles.prm
+
+You can visualize the insertion with Paraview:
+
+.. image:: images/particle_insertion.png
+    :alt: inserted particle at the top of the channel
+    :align: center
+
+The particle has been inserted and it is now possible to simulate its sedimentation.
+
+
+-----------------------
+CFD-DEM Parameter File
+-----------------------
+
+The CFD simulation is carried out using the particle inserted within the previous step. We introduce the different sections of the parameter file ``single-particle-sedimentation.prm`` needed to run this simulation. 
+
+Simulation Control
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The simulation is run for :math:`2` s with a time step of :math:`0.005` s. The time scheme chosen for the simulation is first order backward difference method (BDF1). The simulation control section is shown:
+
+.. code-block:: text
+
+    subsection simulation control
+      set method           = bdf1
+      set output name      = result_
+      set output frequency = 10
+      set time end         = 2
+      set time step        = 0.005
+      set output path      = ./output/
+    end
+
+
+Physical Properties
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We set a density of :math:`996.8` kg/m\ :sup:`3` and a kinematic viscosity of :math:`0.0000008379` m\ :sup:`2`/s as to simulate the particle sedimentation in water at :math:`25^\circ\text{C}` (same conditions as in Ferreira et al. [#ferreira2023]_).
+
+
+.. code-block:: text
+
+    subsection physical properties
+      subsection fluid 0
+        set kinematic viscosity = 0.0000008379
+        set density             = 996.7775
+      end
+    end
+
+
+Boundary Conditions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For the boundary conditions, we choose a slip boundary condition on all the walls of the channel.
+
+.. code-block:: text
```

Comment:
Then you don't need to colorize the boundary condition. You could specify a single boundary as slip and that's it.